pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -196,6 +197,13 @@ class PagerState(
         try {
             animationTargetPage = page
 
+            // pre-jump to nearby item for long jumps as an optimization
+            // the same trick is done in ViewPager2
+            val oldPage = lazyListState.firstVisibleItemIndex
+            if (abs(page - oldPage) > 3) {
```

Comment:
What if I need to jump 15 pages? Should this jump to (target - 3)?

Reply:
I think this code is doing exactly that. so page is the target. oldPage is the current. am I missing something?

Reply:
Oh you're right, totally misread 🚢